packages/esm-admin-openconceptlab-app/src/admin-ocl-card-link.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Layer, ClickableTile } from '@carbon/react';
+import { ArrowRight } from '@carbon/react/icons';
+
+const OpenConceptLabCardLink: React.FC = () => {
+  const { t } = useTranslation();
+  return (
+    <Layer>
```

Comment:
At some point, we probably want to look into turning this into a component so we can reduce the per-app boiler-plate and ensure some kind of consistency.